code/components/nui-resources/src/ResourceUIScripting.cpp

```diff
@@ -425,6 +425,34 @@ static InitFunction initFunction([] ()
 		context.SetResult(nui::HasFocusKeepInput());
 	});
 
+	fx::ScriptEngine::RegisterNativeHandler("SET_NUI_ZINDEX", [] (fx::ScriptContext& context)
+	{
+		fx::OMPtr<IScriptRuntime> runtime;

```

Comment:
Sorry, one more thing. This is a very deep set of nested braces. Can you cut down the depth with some early returns?

Reply:
I changed it